docker-hub/api/latest.yaml

```diff
@@ -0,0 +1,523 @@
+            type: integer
+            example: 3
+          active:
+            description: Number of images counted as active in this repository
+            type: integer
+            example: 2
+          inactive:
+            description: Number of images counted as inactive in this repository
+            type: integer
+            example: 1
+  GetNamespaceRepositoryImagesResponse:
+    description: Paginated list of images in a repository
+    type: object
+    properties:
+      count:
+        description: Total count of images in the repository
+        type: integer
+        example: 100
+      next:
+        description: Link to the next page with the same query parameters if there are more images
+        type: string
+        example: https://hub.docker.com/v2/namespaces/mynamespace/repositories/myrepo/images?&page=4&page_size=20
+        x-nullable: true
+      previous:
+        description: Link to the previous page with the same query parameters if not on first page
+        type: string
+        example: https://hub.docker.com/v2/namespaces/mynamespace/repositories/myrepo/images?&page=2&page_size=20
+        x-nullable: true
+      results:
+        type: array
+        description: Image details
+        items:
+          type: object
+          properties:
+            namespace:
+              description: The repository namespace
+              type: string
+              example: mynamespace
+            repository:
+              description: The repository name
+              type: string
+              example: myrepo
+            digest:
+              description: The image's digest
+              type: string
+              example: sha256:1234567890abcdefghijklmnopqrstuvwxyz1234567890abcdefghijklmnopqr
+            tags:
+              description: The current and historical tags for this image
+              type: array
+              items:
+                type: object
+                properties:
+                  tag:
+                    description: The tag
+                    type: string
+                    example: latest
+                  is_current:
+                    description: |
+                      `true` if the tag currently points to this image
+
+                      `false` if it has been overwritten to point at a different image
+                    type: boolean
+                    example: true
+            last_pushed:
+              description: Time when this image was last pushed
+              type: string
+              x-nullable: true
+              example: '2021-02-24T22:05:27.526308Z'
+            last_pulled:
+              description: Time when this image was last pulled. Note this is updated at most once per hour
+              type: string
+              x-nullable: true
+              example: '2021-02-24T23:16:10.200008Z'
+            status:
+              description: The status of the image based on it's last activity against the active_from time
```

Comment:
```suggestion
              description: The status of the image based on its last activity against the `active_from` time.
```

Reply:
Well this one is just disappointing. Getting my apostrophes wrong... My English teacher would be ashamed